apps/scene2pset/scene2pset.cc

```diff
@@ -230,7 +304,13 @@ main (int argc, char** argv)
 
         /* Triangulate depth map. */
         mve::TriangleMesh::Ptr mesh;
-        mesh = mve::geom::depthmap_triangulate(dm, ci, cam);
+
+        mve::Image<unsigned int> vertex_ids;
+        if (conf.output_correspondence && conf.aabb.empty() && conf.mask.empty())
```

Comment:
Is this `if` necessary? Essentially, obtaining `vertex_ids` is free, no?

Reply:
Essentially free indeed. However, `vertex_ids` will not be used later in the code if this condition doesn't hold.